src/lib/swapper/swappers/CowSwapper/getCowSwapMinMax/getCowSwapMinMax.ts

```diff
@@ -15,17 +17,19 @@ export const getCowSwapMinMax = async (
   deps: CowSwapperDeps,
   sellAsset: Asset,
   buyAsset: Asset,
-): Promise<MinMaxOutput> => {
-  try {
-    const { assetNamespace: sellAssetNamespace } = fromAssetId(sellAsset.assetId)
-    const { chainId: buyAssetChainId } = fromAssetId(buyAsset.assetId)
+): Promise<Result<MinMaxOutput, SwapErrorRight>> => {
+  const { assetNamespace: sellAssetNamespace } = fromAssetId(sellAsset.assetId)
+  const { chainId: buyAssetChainId } = fromAssetId(buyAsset.assetId)
 
-    if (sellAssetNamespace !== 'erc20' || buyAssetChainId !== KnownChainIds.EthereumMainnet) {
-      throw new SwapError('[getCowSwapMinMax]', { code: SwapErrorType.UNSUPPORTED_PAIR })
-    }
+  if (sellAssetNamespace !== 'erc20' || buyAssetChainId !== KnownChainIds.EthereumMainnet) {
+    return Err(
+      makeSwapErrorRight({ message: '[getCowSwapMinMax]', code: SwapErrorType.UNSUPPORTED_PAIR }),
+    )
+  }
 
-    const usdRate = await getUsdRate(deps, sellAsset)
+  const maybeUsdRate = await getUsdRate(deps, sellAsset)
 
+  return maybeUsdRate.map(usdRate => {
```

Comment:
Unchecked monad.

Reply:
Fixed as part of https://github.com/shapeshift/web/commit/a45340d5415b9d8084ee8bb757678f5a9dfad3db